codec/manager.go

```diff
@@ -13,6 +13,8 @@ import (
 )
 
 const (
+	VersionSize = wrappers.ShortLen
```

Comment:
do we want to do VersionLen?

Reply:
I thought about this... I think it makes sense as `Size` because it's used in the `Size` method... 🤷